pkg/collector/corechecks/system/file_handles.go

```diff
@@ -83,7 +83,11 @@ func (c *fhCheck) Run() error {
 	fhInUse := (allocatedFh - allocatedUnusedFh) / maxFh
 	log.Debugf("file handles in use: %f", fhInUse)
```

Comment:
unrelated to your changes: could you remove this debug log line and the others above? They really look like entries added during development and shouldn't be logged from an actual running Agent IMO

Reply:
Sure